frontend/src/cljs/imas_seamap/state_of_knowledge/utils.cljs

```diff
@@ -26,3 +26,21 @@
      :realm                (:name active-realm)
      :province             (:name active-province)
      :ecoregion            (:name active-ecoregion)}))
+
+(defn cql-filter [boundaries]
+  (let [{:keys
+         [network park zone zone-iucn
+          provincial-bioregion mesoscale-bioregion
+          realm province ecoregion]} (boundary-filter-names boundaries)
+        filters (remove
```

Comment:
This makes me want to code-golf it!  It's fine, but the use of both `when` and `remove-nil` makes it look redundant.  I can't think of a clearer way though..